app/assets/javascripts/app/views/comment_mention_view.js

```diff
@@ -0,0 +1,8 @@
+//= require ./publisher/mention_view
+
+app.views.CommentMention = app.views.PublisherMention.extend({
+  initialize: function(opts) {
+    opts.url = Routes.mentionablePost(opts.postId);
+    app.views.PublisherMention.prototype.initialize.call(this, opts);
```

Comment:
Please check with a simple jasmine test that `app.views.PublisherMention.prototype.initialize` is called with the correct url.